lib/dbi/dbi.c

```diff
@@ -26,13 +26,15 @@
 #ifdef WITH_MONGOC
 #include "mongo/mongo-private.h"
 #endif
+#include "json/json-private.h"
 
 int __ogs_dbi_domain;
 
 static ogs_dbi_t *dbi_interfaces[] = {
 #ifdef WITH_MONGOC
     &ogs_dbi_mongo_interface,
 #endif
+    &ogs_dbi_json_interface,
```

Comment:
so how is one or the other selected for use? I don't recall seeing any explanation anywhere.